features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/h2.sql

```diff
@@ -2089,6 +2089,8 @@ CREATE TABLE IF NOT EXISTS AM_SERVICE_CATALOG (
             UPDATED_BY VARCHAR(255),
             ENDPOINT_DEFINITION BLOB NOT NULL,
             METADATA BLOB NOT NULL,
-            PRIMARY KEY (UUID)
+            PRIMARY KEY (UUID),
```

Comment:
changes for other dbs missing

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/9712/commits/eae46b5f9ad76f1794a9642dff0b47904ebedf09